### PR TITLE
Adds examine boxes.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
They're basically fancy rectangles around examine text.

- [x] Do the 200 remaining examine overrides...

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!review

**Review by alsoandanswer — APPROVED** [review]

File: code/game/machinery/doors/firedoor.dm [hunk 1]
```diff
@@ -148,15 +148,15 @@
...
 		return
 
 	if(pdiff >= FIREDOOR_MAX_PRESSURE_DIFF)
-		to_chat(user, SPAN_DANGER("Current pressure differential is [pdiff] kPa. Opening door will likely result in injury."))
+		. += SPAN_DANGER("Current pressure differential is [pdiff] kPa. Opening door will likely result in injury.")
 
-	to_chat(user, "<b>Sensor readings:</b>")
+	. += "<b>Sensor readings:</b>"
 	for(var/index = 1; index <= tile_info.len; index++)
 		var/o = "&nbsp;&nbsp;"
```
Comment (Arrow768) [inline code/game/machinery/doors/firedoor.dm:161]: You should use . here directly instead of first writing to o and then setting . to o

**Review by Arrow768 — CHANGES_REQUESTED** [review]

File: code/modules/cooking/machinery/cooking_machines/_appliance.dm [hunk 2]
```diff
@@ -71,21 +71,20 @@
...
 	if (isemptylist(cooking_objs))
-		to_chat(user, SPAN_NOTICE("It is empty."))
+		. = SPAN_NOTICE("It is empty.")
 		return
 	var/string = "Contains...<ul>"
 	for (var/datum/cooking_item/CI in cooking_objs)
 		string += "<li>\a [CI.container.label(null, CI.combine_target)], [report_progress(CI)]</li>"
 	string += "</ul>"
-	to_chat(user, string)
+	. = string
```
Comment (Arrow768) [inline code/modules/cooking/machinery/cooking_machines/_appliance.dm:87]: Same thing here. -> No need for var/string

File: code/modules/heavy_vehicle/components/_components.dm [hunk 3]
```diff
@@ -47,8 +47,9 @@
 	return
 
 /obj/item/mech_component/proc/return_diagnostics(var/mob/user)
-	to_chat(user, SPAN_NOTICE("[capitalize_first_letters(src.name)]:"))
-	to_chat(user, SPAN_NOTICE(" - Integrity: <b>[round(((max_damage - total_damage) / max_damage) * 100, 0.1)]%</b>" ))
+	. = list()
+	. += SPAN_NOTICE("[capitalize_first_letters(src.name)]:")
+	. += SPAN_NOTICE(" - Integrity: <b>[round(((max_damage - total_damage) / max_damage) * 100, 0.1)]%</b>" )
```
Comment (Arrow768) [inline code/modules/heavy_vehicle/components/_components.dm:52]: I do not think this is going to work.
-> Afaik this is being called by `/mob/living/heavy_vehicle/attackby()` in the mech_interaction.dm
-> It expects that the diagnostic data is printed to chat and not returned as a list (unless you have updated htat?

File: code/modules/integrated_electronics/core/integrated_circuit.dm [hunk 4]
```diff
@@ -10,16 +10,17 @@ a creative player the means to solve many problems.  Circuits are held inside an
...
 	for(var/datum/integrated_io/O in outputs)
 		if(O.linked.len)
-			to_chat(user, "The '[O]' is connected to [O.get_linked_to_desc()].")
+			. += "The '[O]' is connected to [O.get_linked_to_desc()]."
 	for(var/datum/integrated_io/activate/A in activators)
 		if(A.linked.len)
-			to_chat(user, "The '[A]' is connected to [A.get_linked_to_desc()].")
+			. += "The '[A]' is connected to [A.get_linked_to_desc()]."
 	any_examine(user)
 	interact(user)
```
Comment (Arrow768) [inline code/modules/integrated_electronics/core/integrated_circuit.dm:25]: At a quick glance it does not seem like internal_examine is being called anywhere?

File: code/modules/organs/subtypes/machine.dm [hunk 5]
```diff
@@ -230,12 +230,12 @@
...
 	. = ..()
 
-/obj/item/organ/internal/ipc_tag/examine(mob/user)
+/obj/item/organ/internal/ipc_tag/get_examine_text(mob/user, distance, is_adjacent, infix, suffix)
 	. = ..()
-	to_chat(user, SPAN_NOTICE("Serial Autogeneration: [auto_generate ? "Yes" : "No"]"))
-	to_chat(user, SPAN_NOTICE("Serial Number: [serial_number]"))
-	to_chat(user, SPAN_NOTICE("Ownership Info: [ownership_info]"))
-	to_chat(user, SPAN_NOTICE("Citizenship Info: [citizenship_info]"))
+	. +=  SPAN_NOTICE("Serial Autogeneration: [auto_generate ? "Yes" : "No"]")
```
Comment (Arrow768) [inline code/modules/organs/subtypes/machine.dm:235]:
```suggestion
	. += SPAN_NOTICE("Serial Autogeneration: [auto_generate ? "Yes" : "No"]")
```

**Review by Arrow768 — CHANGES_REQUESTED** [review]

File: code/modules/integrated_electronics/core/integrated_circuit.dm (hunk 4 above)
Comment (NonQueueingMatt) [reply code/modules/integrated_electronics/core/integrated_circuit.dm:25]: apparently not?

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**